pkg/ccl/backupccl/backup_intents_test.go

```diff
@@ -120,5 +119,12 @@ func TestCleanupIntentsDuringBackupPerformanceRegression(t *testing.T) {
 		// Each of the 1,000 transactions is expected to get pushed once, but in an
 		// actual run of the test we might see more pushes (e.g. of other transactions).
 		require.GreaterOrEqual(t, 1100, int(numPushBatches.Load()))
+
+		if !abort {
```

Comment:
for my own clarity, we needed to move this in case it bumped numIntentResolveBatches or numPushBatches?

Reply:
Yes, potentially, though not expected in the general case. I just tried to tighten the counting to include only the actual backup as much as possible.